ironfish/src/rpc/routes/chain/importChain.ts

```diff
@@ -74,7 +75,7 @@ function deserializeChunk(node: IronfishNode, reader: bufio.BufferReader): Block
   for (let i = 0; i < blocksLength; i++) {
     const blockReader = bufio.read(reader.readVarBytes(), true)
     const serializedBlock = readBlock(blockReader)
-    const block = node.chain.strategy.blockSerde.deserialize(serializedBlock)
+    const block = BlockSerde.deserialize(serializedBlock)
```

Comment:
I think this RPC should be deleted now, right? It can happen after this PR.

Reply:
I'll delete these rpc routes and unneeded snapshot code in a subsequent PR